File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int>ans;
        int b=nums.size();
        for(int i=0;i<b-1;i++)
        {
            for(int j=i+1;j<b;j++)
            
                if(nums[i]+nums[j]==target)
                {
                    ans.push_back(i);
                    ans.push_back(j);
                    break;
                    
                }
            }
            
             return ans;

        }
};

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int length=nums.length;
        Map<Integer,Integer> m1=new HashMap<>();
        for(int i=0;i<length;i++)
        {
            if(m1.containsKey(target-nums[i]))
            {
                return new int[]{i,m1.get(target-nums[i])};
            }
            m1.put(nums[i],i);
        }

        return new int[]{};
        
    }
}